.github/CODEOWNERS

```diff
@@ -0,0 +1,81 @@
+# Describes which individuals/teams are reponsible for which code in the
+# repository.
+#
+# NOTE: ownership by an individual is a smell, as is ownership by multiple
+# teams. We have a soft goal of arranging our code so that each entry names
+# a single team as owner. Better not to hide the truth, though; we'd rather
+# be honest about where we still have individuals as owners.
+#
+# See: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+/.cargo                             @benesch
+/.config                            @benesch
+/.devcontainer                      @mjibson
+/.github                            @benesch
+/bin
+/ci                                 @MaterializeInc/qa
+/doc/user                           @MaterializeInc/DevEx
+/doc/developer/reference/compute    @MaterializeInc/compute
+/doc/developer/reference/storage    @MaterializeInc/storage
+/src/adapter                        @MaterializeInc/surfaces
```

Comment:
```suggestion
/doc/user                           @MaterializeInc/devex
/doc/developer/reference/compute    @MaterializeInc/compute
/doc/developer/reference/storage    @MaterializeInc/storage
/misc/dbt-materialize               @MaterializeInc/devex
/src/adapter                        @MaterializeInc/surfaces
```

Reply:
Adding `/misc/dbt-materialize` and defaulting to using the team slug (`devex`).

Reply:
Thanks, blended.